frontend/src/routes/project/[project_code]/+page.svelte

```diff
@@ -8,93 +8,141 @@
 	import t from '$lib/i18n';
 	import type { PageData } from './$types';
 	import DeleteModal from '$lib/components/modals/DeleteModal.svelte';
-	import { _deleteProjectUser, type ProjectUser } from './+page';
+	import {
+		_changeProjectDescription,
+		_changeProjectName,
+		_deleteProjectUser,
+		type ProjectUser,
+	} from './+page';
```

Comment:
this looks extremely odd to me... I'd be inclined toward keeping those api's outside of the endpoint file since I would expect to only see verb bindings in the `+page.ts`

Reply:
maybe we could chat more if you have time but the only connection between these two files should be the "page data":  https://kit.svelte.dev/docs/load

Reply:
I've added it to discuss as a team today on what we want to do.